src/ui/public/courier/saved_object/saved_object.js

```diff
@@ -302,14 +287,14 @@ export function SavedObjectProvider(esAdmin, kbnIndex, Promise, Private, Notifie
      * @resolved {String} - The id of the doc
```

Comment:
Goot catch. It doesn't appear to have ever returned the id. `doIndex` is defined [here](https://github.com/elastic/kibana/blob/master/src/ui/public/courier/data_source/_abstract_doc_source.js#L69), which calls sendToEs and resolves [here](https://github.com/elastic/kibana/blob/master/src/ui/public/courier/data_source/_doc_send_to_es.js#L49) 

Updated the comment